### PR TITLE
Add Atom and Jupyter setup instructions work on Phase Field landing page

### PR DESCRIPTION
**Pull request by dschwen**
This has some small changes to `moose.css` @aeslaughter (to render bold text and bring back bullets)

Refs #8368

### CODE REVIEW

**Comment by aeslaughter** [comment]
Let me know when you are done adding content to this PR, I don't want to start reviewing if you plan to keep adding commits. Thanks.

**Comment by dschwen** [comment]
@aeslaughter I'm trying to fix the docs build errors, but it seems like the docs system does not pick up any mesh classes! Are you working from home? Can you come into the gitter chat?

File: docs/content/documentation/modules/phase_field/ICs/EBSD.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+[Mesh]
+  type = EBSDMesh
+  filename = IN100_128x128.txt
+  uniform_refine = 4
+[]
+```
+
+## EBSD Reader UserObject
+
+The user object reads in the data file, using the name supplied in the mesh block,
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/ICs/EBSD.md]: UserObject?

File: docs/content/documentation/modules/phase_field/ICs/EBSD.md [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+and stores a data object with the local data at each material point as well as the
+average data about each grain. The block syntax is very simple:
+
+```text
+  [./ebsd]
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/ICs/EBSD.md]: I think you should include the [UserObjects] block as well so people no where to put this.

File: docs/content/documentation/modules/phase_field/ICs/EBSD.md [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+[]
+
+[ICs]
+  [./PolycrystalICs]
+    [./ReconVarIC] #Uses the data from the user object to initialize the variables for all the order parameters.
+      ebsd_reader = ebsd #Name of the ebsd reader user object
+    [../]
+  [../]
+[]
+```
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/ICs/EBSD.md]: Let it be known that I would prefer that this was imported from an actual input file, but you can do what you want.

File: docs/content/documentation/modules/phase_field/ICs/EBSD.md [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+_euler_angles(0) = d.phi1;
+_euler_angles(1) = d.phi;
+_euler_angles(2) = d.phi2;
+```
+
+The EBSDReader average grain data is extracted using the `getAvgData(unsigned int)`
+function call, where you pass in the grain number for which you want the data.
+The available data that can be extracted
+
+* `phi1` - The average first Euler angle $$\phi_1$$
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/ICs/EBSD.md]: Use a single $ to make this inline

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+# Analytical Solution for a 1D equilibrium interface
+
+!image docs/media/phase_field/kks_c.png width=200px padding-left=20px float=right caption=Composition $c$ for 2D simulation domain
+
+!image docs/media/phase_field/kks_example_split_eta_0050.png width=200px padding-left=20px float=right clear=both caption=Order parameter $\eta$ along $y=0$
+
+!image docs/media/phase_field/kks_example_split_c_0050.png width=200px padding-left=20px float=right clear=both caption=Composition $c$ along $y=0$
+
+!text modules/phase_field/examples/kim-kim-suzuki/kks_example_noflux.i max-height=300px strip-extra-newlines=True overflow-y=scroll
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md]: The input files seem like they should go after the text.

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+
+$$
+\eta_0(x) = \frac{1}{2} \left[1-\tanh{\left( \frac{\sqrt{w}}{\sqrt{2} \epsilon} x\right)} \right]
+$$
+
+$$
+c_0(x) =  h(\eta_0(x))c_S^e + [1-h(\eta_0(x))]c_L^e
+$$
+
+where we use the switching function $h(\eta) = \eta^3(6\eta^2-15\eta+10)$, the gradient energy coefficient $\epsilon^2 = 1$ and the barrier function height $w=1$. (Note that there is a typo in Eq. (49) of the paper, $\epsilon$ should be in the denominator of the argument to the $\tanh$ function, not $\sqrt{\epsilon}$.) In the following example, we equilibrate a flat interface between a solid phase ($\eta = 1$, $f^S = (c_S-c_S^e)^2$, $c_S^e = 0.9$) and a liquid phase ($\eta = 0$, $f^L = (c_L-c_L^e)^2$, $c_L^e = 0.1$) in a 2D simulation. The vector postprocessor `LineValueSampler` is used to obtain the values of $\eta$ and $c$ along $y=0$, the results are output to a CSV file, and plotted together with the 1D analytical solution. (We will use no-flux boundary conditions, so a boundary conditions block is not required in the input file.)
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md]: "the paper" should be an actual ref. using bibtex

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+
+!text modules/phase_field/examples/kim-kim-suzuki/kks_example_dirichlet.i max-height=300px strip-extra-newlines=True overflow-y=scroll
+
+To perform a more quantitative comparison of the simulation results to the analytical solution, we will calculate the $L^2$ norm of the difference between the simulation result and the analytical solution. The $L^2$ norm is defined for this case as
+
+$$
+\left|u - u_h\right|_{L^2} = \left( \int_\Omega (u - u_h)^2 d\Omega \right)^{1/2}
+$$
+
+where $u$ is the analytical solution, $u_h$ is the equilibrium solution from simulation, and $\Omega$ is the domain. The $L^2$ norm can be obtained in the MOOSE framework using the `ElementL2Error` postprocessor. It can be shown from the properties of the finite element method that for the linear Lagrange elements used in the split formulation,
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md]:
```
[/ElementL2Error.md](ElementL2Error)
```

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+
+$$
+|u - u_h|_{L^2} \le Ch^2
+$$
+
+where $h$ is the characteristic element size (for 2D square elements, $h$ is the length of one side) and $C$ is a constant specific to the problem. Thus, as the mesh is refined, $L^2$ error should decrease (at least) quadratically with $h$.
+
+(In performing this comparison between the analytical solution and simulation results, if a no-flux boundary condition is used in the simulation, the order parameter and composition profiles may shift slightly in the $+x$ or $-x$ direction, even if the analytical solution is used as an initial condition. This makes it difficult to compare to an analytical solution centered at $x=0$. Therefore, we simulate the $x \ge 0$ half of the domain and use a Dirichlet boundary condition of $\eta=0.5$ and $c=0.5$ at $x=0$, which prevents the interface from moving. For the verification, the size of the domain was also reduced to lower the computational cost.)
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Analytical.md]: This probably doesn't need to be in parentheses.

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md [hunk 9]
```diff
@@ -0,0 +1,362 @@
...
+$$
+J= \phi_j \frac{\partial R}{\partial c} = -\phi_j
+$$
+For $\eta$
+$$
+J= \phi_j \frac{\partial R}{\partial \eta} = \phi_j \frac{dh}{d\eta}(c_b-c_a)
+$$
+
+## Bibliography
+* [KKS] S.G. Kim, W.T. Kim, and T. Suzuki, Phys. Rev. E 60 (1999) 7186
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md]: Utilize the bibtex feature. http://www.mooseframework.org/docs/moose_docs/site/utilities/documentation/moose_flavored_markdown/index.html

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md [hunk 10]
```diff
@@ -0,0 +1,362 @@
...
+
+The KKS system uses multiple MOOSE materials to provide values for free energy functions, the switching function $h(\eta)$, and the double well function $g(\eta)$.
+Providing these as materials allows the functions to be bundled in a single place, while being used by multiple kernels. Furthermore the automatic differentiation feature
+used in the parsed function materials provides the necessary derivatives at no cost to the developer. The derivatives are stored in material properties and follow a naming scheme
+that is defined in ```KKSBaseMaterial.C```.
+
+# Cahn-Hilliard Kernels
+
+## KKSSplitCHCRes
+This is the split version. In this kernel, we calculate the chemical potential $\mu$ from $\frac{\partial F}{\partial c}$. The non-linear variable for this Kernel is the concentration $c$. To calculate $\frac{\partial c}{\partial t}$ and $\nabla \cdot M(c) \nabla \mu$, we use the `CoupledTimeDerivative` and `SplitCHWRes` kernels, respectively, as described [here](phase_field/Phase Field Equations.md).
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md]: You have references to objects throughout this document, they should probably link to the markdown files.

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md [hunk 11]
```diff
@@ -0,0 +1,362 @@
+# KKS Model Definitions
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md]: I hope your math is right, there is no way I can check it.

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+
+## See also
+
+* [Derivation of the KKS system residuals and Jacobians](Multi Phase/KKS Derivations.md)
+* [Comparison with the analytical solution for an equilibrium interface](Multi Phase/KKS Analytical.md) for a simple 2-component example of the KKS model.
+* [KKS phase-field model with 3 or more components](Multi Phase/KKS Multi Component Example.md)
+
+## References
+
+1. Kim et al., Phys. Rev. E, 60 (1999) 7186-7197
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS.md]: Use bibtex

File: docs/content/documentation/modules/phase_field/Multi Phase/WBM.md [hunk 13]
```diff
@@ -0,0 +1,108 @@
+# Multiphase models
+
+Material objects that internally derive from ```DerivativeFunctionMaterialBase``` ([Doxygen](http://mooseframework.org/docs/doxygen/modules/classDerivativeFunctionMaterialBase.html)), like the materials for the [Parsed Function Kernels](ParsedFunctionKernels) are used to provide the free energy expressions for each phase.
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/WBM.md]: Parsed Function Kernels link is not right

File: docs/content/documentation/modules/phase_field/Multi Phase/WBM.md [hunk 14]
```diff
@@ -0,0 +1,108 @@
...
+
+The flexible **multiphase model** uses _n_ order parameters to control _n_ phases while employing a Lagrange multiplier based constraint to enforce the sum of all phase contributions to be one at every point in the simulation cell.
+
+For multiphase models with _n_ phases ```DerivativeMultiPhaseMaterial``` ([Doxygen](http://mooseframework.org/docs/doxygen/modules/classDerivativeMultiPhaseMaterial.html)) can be used to form the global free energy as
+
+$$
+F = \left[ \sum_i^n h_i(\eta_i)F_i \right] + g(\vec\eta)
+$$
+
+!parameters /Materials/DerivativeMultiPhaseMaterial
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/WBM.md]: This seems strange just plopped in the middle of the text

File: docs/content/documentation/modules/phase_field/Multi Phase/WBM.md [hunk 15]
```diff
@@ -0,0 +1,108 @@
...
+
+## Penalty constraint
+
+As an alternative (softer) constraint enforcement we provide the ```SwitchingFunctionPenalty``` ([Doxygen](http://mooseframework.org/docs/doxygen/modules/classSwitchingFunctionPenalty.html)) kernel, which effectively adds a free energy penalty of $\gamma k(\vec \eta)^2$ (with $\epsilon=0$), where $\gamma$ is the penalty prefactor (```penalty```). The constraint is enforced approximately to a tolerance of $\frac1\gamma$ (depending on the shape and units of the free energy).
+
+The gradient interface energy term for multiphase models with _n_>2 is derived [here](ACMultiInterface) and provided by the ```ACMultiInterface``` kernel.
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/WBM.md]: here link is not correct

File: docs/content/documentation/modules/phase_field/Multi Phase/WBM.md [hunk 16]
```diff
@@ -0,0 +1,108 @@
...
+  fb_name = Fb  # f_name of the phase B free energy function
+  args = 'c'
+  eta = eta     # order parameter that switches between A and B phase
+  third_derivatives = false
+  outputs = exodus
+[../]
+[]
+```
+
+Note that the phase free energies are single wells. The global free energy landscape will however have a double well character in this example.
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Multi Phase/WBM.md]: Should this be a admonition

```
!!! note
    blah blah...
```

File: docs/content/documentation/modules/phase_field/Solving.md [hunk 17]
```diff
@@ -4,7 +4,7 @@ Once you have developed a phase field model using implicit time integration, you
 
 ## Solution Methods
 
-Three solve methods are available in MOOSE, which are set in the [Executioner](../../../framework/systems/Executioner/Overview) block using the _solve_type_ parameter, where the three possible options are:
+Three solve methods are available in MOOSE, which are set in the [Executioner](Executioner/index.md) block using the _solve_type_ parameter, where the three possible options are:
 
 * `NEWTON` - Direct solution of the system of equations using Newton's method. The full and accurate Jacobian is required. Thus, the [Preconditioner](../../../framework/systems/Preconditioning/Overview) block must be employed for systems with multiple nonlinear variables.
```
Comment (aeslaughter) [inline docs/content/documentation/modules/phase_field/Solving.md]: [Preconditioner] link needs update

**Review by aeslaughter — CHANGES_REQUESTED** [review]

**Comment by aeslaughter** [comment]
I went through this rather quickly and found some basic things, I won't be much help checking your math. Also, you include a lot of input file syntax that I think should be updated to reference actual input files, but this is your code you can do what you want.

File: docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md (hunk 11 above)
Comment (dschwen) [inline docs/content/documentation/modules/phase_field/Multi Phase/KKS Derivations.md]: Trust me, I know what I'm...

nevermind.

Jacobian check says that at least the second half is correct :-)

**Review by aeslaughter — APPROVED** [review]

**Pull request merged**